benchmarking/example_classification.py

```diff
@@ -10,33 +11,37 @@
 from sklearn.metrics import roc_auc_score
 from sklearn.preprocessing import MinMaxScaler
 
-from skfp import (
-    AtomPairFingerprint,
-    ECFPFingerprint,
-    ERGFingerprint,
-    MACCSKeysFingerprint,
-    MAP4Fingerprint,
-    MHFPFingerprint,
-    TopologicalTorsionFingerprint,
-)
-
-dataset_names = [
-    "ogbg-molhiv",
-    "ogbg-molbace",
-    "ogbg-molbbbp",
+import skfp.fingerprints
+
+fingerprint_classes = [
+    cls
+    for name, cls in inspect.getmembers(skfp.fingerprints, predicate=inspect.isclass)
 ]
-property_names = [
-    "HIV_active",
-    "Class",
-    "p_np",
+
+SIZE = 125
+
+dataset_params = [
+    ("ogbg-molhiv", "HIV_active"),
+    ("ogbg-molbace", "Class"),
+    ("ogbg-molbbbp", "p_np"),
 ]
 
-for dataset_name, property_name in zip(dataset_names, property_names):
+
+classifier_parameters = [
+    (RandomForestClassifier, {"n_jobs": -1, "class_weight": "balanced"}),
+    (
+        LogisticRegression,
+        {"class_weight": "balanced", "penalty": None, "n_jobs": -1},
+    ),
+    (lgb.LGBMClassifier, {"n_jobs": -1, "class_weight": "balanced", "verbose": -1}),
+]
+
+for dataset_name, property_name in dataset_params:
     dataset = GraphPropPredDataset(name=dataset_name, root="../dataset")
     split_idx = dataset.get_idx_split()
-    train_idx = np.array(split_idx["train"])
-    valid_idx = np.array(split_idx["valid"])
-    test_idx = np.array(split_idx["test"])
+    train_idx = np.array(split_idx["train"])[:SIZE]
+    valid_idx = np.array(split_idx["valid"])[:SIZE]
+    test_idx = np.array(split_idx["test"])[:SIZE]
```

Comment:
I think that we shouldn't limit the sizes here. Obviously, some fingerprints will be quite slow, but if we want to get classification results with them, this is the price.

Reply:
That is our goal in the end. However, testing it with >100 molecules already takes a few minutes. This was just for having shorted feedback loop if it even works

Reply:
Alright